packages/frontend/src/components/DrawerMenu/DrawerMenu.styled.ts

```diff
@@ -32,6 +33,7 @@ export const NavStyled = styled.nav<NavStyledProps>`
   padding: 20px;
   background-color: ${Color.Black};
   position: fixed;
+  z-index: ${zIndex.DrawerMenu};
   transform: ${({ isOpen }) => (isOpen ? "transform: translateX(0)" : "translateX(100%)")};
```

Comment:
co tu się zadziało i czemu to działa !!!!! jeżeli isOpen jest true to do transform przypisujesz transform: translateX(0)
i wychodzi ci transform: transform: translateX(0) zmień to